pull_request_template.md

```diff
@@ -0,0 +1,4 @@
+#[[issue Nr]] << Descriptive message >>
```

Comment:
I'd suggest we have "closes #[[issue Nr]]" instead

Reply:
> I'd suggest we have "closes #[[issue Nr]]" instead

I think maybe we will get the same result from these two code.